lib/models/solid-host.js

```diff
@@ -109,4 +111,9 @@ class SolidHost {
   }
 }
 
+function getHostName (origin) {
```

Comment:
Doesn't using `origin` confuse that there may be other URIs that the hostname is extracted from than the `Origin`? Or am I confused for other reasons?

Reply:
You are right, I will change the parameter name.